Minor syntax edits to Home page. (#35)
thegooddocsproject/website-hugo: content/en/roles/index.md

    *   Inviting active and eligible community members into relevant roles, based on our eligibility criteria.
    *   Moving people who have been inactive for a year into a retired status, with thanks.
    *   Selecting a PSC chair.
*   Ad-hoc invitations may be initiated throughout the year, as needed.
*   Retired members should expect to be welcomed back if they become active again.
*   PSC discussions about membership will typically be conducted privately in order to respect the feelings of those being discussed.
*   PSC selection decisions will follow our decision making process.

## Invitation email

_This example email can be customized to be made personal_.

Hi \[Name],

We, the Project Steering Committee of The Good Docs Project, have been impressed with your contributions to The Good Docs Project, and would like to invite you to take up our official role of \[Role].

To accept, could you please reply publicly to this email list confirming:
1. You would like to accept the role.
1. You feel you meet the eligibility criteria, including your intent to continue participating in this project for a while.

> 1. You accept the responsibilities associated with the role, as described in https://thegooddocsproject.dev/roles/.

Author: ```suggestion
1. You accept the responsibilities associated with the role, as described on the [Roles](/roles) page.
```

> 1. You accept the responsibilities associated with the role, as described in https://thegooddocsproject.dev/roles.html.

Author: ```suggestion
1. You accept the responsibilities associated with the role, as described in https://thegooddocsproject.dev/roles/.
```

Regards,

\[Name], on behalf of the PSC

## Next steps
* Wait untill the person accepts the invitation.

> * Add the person's name to the [/who-we-are](/who-we-are) page.

Author: ```suggestion
* Add the person's name to the [Who We Are](/who-we-are) page.
```
